packages/fxa-event-broker/src/queueworker/queueworker.service.spec.ts

```diff
@@ -250,6 +262,29 @@ describe('QueueworkerService', () => {
         'resource-server-client-id',
       ]);
     });
+    
+    it('handles apple migration event', async () => {
+      const msg = updateStubMessage(appleMigrationMessage);
+      await (service as any).handleMessage(msg);
+      
+      // topic prefix is undefined and clientId is always Pocket
+      const topicName = 'undefined749818d3f2e7857f';
```

Comment:
Why is the topic prefix undefined? Shouldn't the value just come from the config?